android/java/org/chromium/chrome/browser/ntp_background_images/RewardsBottomSheetDialogFragment.java

```diff
@@ -135,7 +136,9 @@ public void onClick(View v) {
             turnOnAdsButton.setOnClickListener(new View.OnClickListener() {
                 @Override
                 public void onClick(View v) {
-                    turnOnAds();
+                    if (BraveActivity.getBraveActivity() != null) {
+                        BraveActivity.getBraveActivity().openRewardsPanel();
```

Comment:
@deeppandya is the new requirement that we open the rewards panel where users can join rewards? Thanks

Reply:
@tmancey with this PR, we need to ask users to declare geo when they create wallet for rewards. so it's handled by the UI we have on rewards panel.